src/engine/ReactPlugin/ToolReactPlugin.js

```diff
@@ -40,7 +55,15 @@ export function withToolPlugin(ReactComponent, methods = []) {
       };
     }, [off, on, profile.name]);
 
-    return <RefComponent {...props} ref={ref} />;
+    return (
+      <div
```

Comment:
This div is causing an issue on the **Get Started** tab, it's not filling the component.
**Previously*
![image](https://github.com/user-attachments/assets/c0e4ee09-32ae-4bd3-a8f5-19d8011e88c9)
**Now**
![image](https://github.com/user-attachments/assets/f40e5318-459d-4368-97b6-e8c9a9a438b5)

Reply:
Oh damn, I didn't see that. 
The logs were actually there, but the div was not visible. Added the 100% height to the new container div. 

Nice catch